src/main.rs

```diff
@@ -51,11 +52,17 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 		/// The number of pages to prerender surrounding the currently-shown page; 0 means no
 		/// limit. By default, there is no limit.
 		optional -p,--prerender prerender: usize
+		/// Custom white and black colors
+		optional -w,--white-color white: String
+		optional -b,--black-color black: String
```

Comment:
```suggestion
		/// Custom white color
		optional -w,--white-color white: String
		/// Custom black color
		optional -b,--black-color black: String
```

Just so it shows up clearer on help text

Reply:
Actually, could you also specify the format of how to pass it in - e.g. `0xffffff` vs `ffffff` vs `#ffffff` - which one to use?